homework12/task1.2.py

```diff
@@ -0,0 +1,12 @@
+import time
+def my_func(func):
+    def my_function():
+        print("function start time, and function name:")
```

Comment:
the decorator should show the time when the function is called, I don't see it there